app/collections/create-form.jsx

```diff
@@ -77,6 +89,18 @@ class CollectionsCreateForm extends React.Component {
             placeholder="Collection Name"
           />
         </label>
+        <label>
```

Comment:
This doesn't actually have a text label, so I'm not sure of the utility of wrapping the input with one. Same with the name input. We should add labels or remove the label tags.